platform/platform-filter-response/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,10 +26,13 @@
         <AD description="The HTTP X-XSS-Protection response header is a feature that stops pages from loading when they detect reflected cross-site scripting (XSS) attacks."
             name="X-XSS-Protection" id="xXssProtection" required="true" type="String"
             default="1; mode=block"/>
+        <AD description="The Cache-Control header specifies the behavior for browsers and proxies to cache web resources."
```

Comment:
I thought this configuration would be more generic where it would be a list of `header=value` that should be injected into all responses.